README.md

```diff
@@ -48,59 +49,168 @@ For .NET Core CLI, use the following command:
 Install-Package TagShelf.Alfred.ApiWrapper
 ```
 
-## Usage
+# Environment
```

Comment:
If the end-user will only have access to only one environment, I think we shouldn't even include this section in the README. I would add to the requirements the following (can be paraphrased):

Access to the frontend URL at https://app.tagshelf.com/

Reply:
I left it in because the environment is always going to be referenced in the code when they use it since they have to authenticate the calls made to the library so might as well leave them the description of the production environment. Also the description of it was something Raul requested Eduardo B. in the past.